File: app/src/main/java/com/texon/engineeringsmartbook/View/BookEnrollment.java
package com.texon.engineeringsmartbook.View;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.ImageButton;

import com.texon.engineeringsmartbook.R;
import com.texon.engineeringsmartbook.View.Auth.Login;

public class BookEnrollment extends AppCompatActivity {

    ImageButton btnBackMenu, btnNotificationsMenu, bntHomeMenu, btnProfileMenu, btnScanMenu;
    Button btnApply;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_book_enrollment);

        btnBackMenu = findViewById(R.id.btnBackMenu);
        btnNotificationsMenu = findViewById(R.id.btnNotificationsMenu);
        bntHomeMenu = findViewById(R.id.btnHomeMenu);
        btnProfileMenu = findViewById(R.id.btnProfileMenu);
        btnScanMenu = findViewById(R.id.btnScannerMenu);

        btnApply = findViewById(R.id.btnApply);


        btnBackMenu.setOnClickListener(v -> {
            Intent intent = new Intent(getApplicationContext(), Login.class);
            startActivity(intent);
            finish();
        });

        btnApply.setOnClickListener(v -> {
            Intent intent = new Intent(getApplicationContext(), SuccessfullyEnrolled.class);
            startActivity(intent);
            finish();
        });
    }

    @Override
    public void onBackPressed() {
        Intent intent = new Intent(getApplicationContext(), Login.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/texon/engineeringsmartbook/ui/main/view/BookEnrollment.kt
package com.texon.engineeringsmartbook.ui.main.view

import android.content.Intent
import android.os.Bundle
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity
import com.texon.engineeringsmartbook.databinding.ActivityBookEnrollmentBinding
import com.texon.engineeringsmartbook.ui.main.view.auth.Login
import kotlinx.coroutines.DelicateCoroutinesApi

@DelicateCoroutinesApi
class BookEnrollment : AppCompatActivity() {

    private lateinit var binding: ActivityBookEnrollmentBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityBookEnrollmentBinding.inflate(layoutInflater)
        setContentView(binding.root)
        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN)

        binding.titleBar.btnBackMenu.setOnClickListener{
            val intent = Intent(applicationContext, Login::class.java)
            startActivity(intent)
            finish()
        }

        binding.btnApply.setOnClickListener {
            val intent = Intent(applicationContext, SuccessfullyEnrolled::class.java)
            startActivity(intent)
            finish()
        }

        binding.appBar.btnProfileMenu.setOnClickListener {  }

    }

    override fun onBackPressed() {
        val intent = Intent(applicationContext, Login::class.java)
        startActivity(intent)
        finish()
    }
}